packages/design-tokens/stories/Typography.stories.tsx

```diff
@@ -1,26 +1,27 @@
+/* eslint-disable no-restricted-globals */
```

Comment:
Lint fixes and ignoring no-restriced-globals so we can get the CSS variables from the stylesheet for typography CSS variables